plugin/src/test/resources/multiple_datasources.json

```diff
@@ -11,7 +11,7 @@
   },
   {
     "name" : "prometheus-1",
-    "connector": "prometheus",
+    "connector": "PROMETHEUS",
```

Comment:
Case sensitive systems could have trouble with this change, seems like it a breaking change.

Reply:
I had to change this because GSON couldn't recognize the lower-case version since it wasn't the exact same as the enum value.

Reply:
Checkout https://stackoverflow.com/a/18343576/533057, seems like there are a couple of options to ensure the output is backwards compatible, name annotations or using a case-insensitive TypeAdapterFactory.